openquakeplatform/openquakeplatform/static/js/irv_viewer.js

```diff
@@ -105,8 +105,16 @@ function createIndex(la, index) {
     return indicator;
 }
 
-function combineIndicators(nameLookUp, themeObj, JSONthemes) {
```

Comment:
Normally, an explicit parameter has more priority than an implicit one.
Please, remove testSessionProjectDef from function definition and use arguments[3] and, if present, set it as projectDef inverting the logic.